core-rust/mesh-api-server/src/mesh_api/conversions/operations.rs

```diff
@@ -140,3 +186,129 @@ fn resolve_global_fee_balance_changes(
     }
     Ok(fee_balance_changes)
 }
+
+/// This method converts Transaction V1 instructons to operations.
+/// Note that it supports very limited number of instructions because it
+/// is implemented just for sanity checks.
```

Comment:
Thanks for the comment :) - although I'm not sure we can say it's just for sanity checks - we're exposing this in the API and clients could use it as they want. That said, I imagine it's mostly used by clients for sanity checks of constructed transactions.

I think a clearer comment would be that this parsing only detects operations in the form they are created in the construction API, and can miss or return unexpected results for other transactions.

Perhaps we should call this method `limited_parse_mesh_api_operations_from_v1_instructions` or something?